### PR TITLE
change ancient tests to correct size

### PR DESCRIPTION
**Pull request by jeffwashington**
#### Problem
fix tests to be more realistic

#### Summary of Changes
allocate correctly sized append vecs

Fixes #

### CODE REVIEW

File: runtime/src/ancient_append_vecs.rs [hunk 1]
```diff
@@ -1155,62 +1156,76 @@ pub mod tests {
...
+                    );
 
-                if num_slots > 0 {
-                    let mut result = db
-                        .write_ancient_accounts(1, accounts_to_write)
-                        .shrinks_in_progress;
-                    let one = result.drain().collect::<Vec<_>>();
-                    assert_eq!(1, one.len());
-                    assert_eq!(target_slot, one.first().unwrap().0);
+                    let bytes = storages
```
Comment (apfitzge) [inline runtime/src/ancient_append_vecs.rs:1191]: nice added check!

File: runtime/src/ancient_append_vecs.rs [hunk 2]
```diff
@@ -1155,62 +1156,76 @@ pub mod tests {
...
     #[test]
     fn test_write_ancient_accounts() {
-        for num_slots in 0..4 {
-            for combine_into in 0..=num_slots {
-                if combine_into == num_slots && num_slots > 0 {
-                    // invalid combination when num_slots > 0, but required to hit num_slots=0, combine_into=0
-                    continue;
-                }
-                let (db, storages, slots, _infos) = get_sample_storages(num_slots);
+        for data_size in [None, Some(10_000_000)] {
```
Comment (jeffwashington) [inline runtime/src/ancient_append_vecs.rs:1159]: this is a massive diff due to indentation. removing whitespace shows the interesting changes.

File: runtime/src/accounts_db.rs [hunk 3]
```diff
@@ -10469,9 +10469,9 @@ pub mod tests {
         account_data_size: Option<u64>,
     ) -> Arc<AccountStorageEntry> {
         let (_temp_dirs, paths) = get_temp_accounts_paths(1).unwrap();
-        let size: usize = 123;
+        let size: usize = aligned_stored_size(account_data_size.unwrap_or(123) as usize);
         let mut data = AccountStorageEntry::new(&paths[0], slot, id, size as u64);
-        let av = AppendVec::new(&tf.path, true, 1024 * 1024);
+        let av = AppendVec::new(&tf.path, true, (1024 * 1024).max(size));
```
Comment (apfitzge) [inline runtime/src/accounts_db.rs:10474]: what's the reason for imposing a minimum appendvec size here?

**Review by apfitzge — APPROVED** [review]
lgtm - added a question

File: runtime/src/accounts_db.rs (hunk 3 above)
Comment (jeffwashington) [reply runtime/src/accounts_db.rs:10474]: cause that's how it's been. Just making sure we get an append vec of big enough size so tests pass. Could probably be made much smaller. in my case, I needed bigger than this. I can look into changing this in a follow on pr to fix the status quo.

**Pull request merged**